checker/checker.js

```diff
@@ -291,7 +291,7 @@ function action_required (ow, args) {
       conclusion: 'action_required',
       details_url: 'http://opensource.adobe.com/cla.html',
       title: 'Sign the Adobe CLA!',
-      summary: 'No signed agreements were found. Please [sign the Adobe CLA](http://opensource.adobe.com/cla.html)! Once signed, close and re-open your pull request to run the check again.\n\n If you are an Adobe employee, you do not have to sign the CLA. Instead contact Adobe\'s Open Source Office about the failure by mentioning them on the pull request with **@adobe/open-source-office** or via email <grp-opensourceoffice@adobe.com>.'
+      summary: 'No signed agreements were found. Please [sign the Adobe CLA](http://opensource.adobe.com/cla.html)! Once signed, close and re-open your pull request to run the check again.\n\n If you are an Adobe employee, you do not have to sign the CLA. Instead [add yourself to this GitHub Org](https://git.corp.adobe.com/OpenSourceAdvisoryBoard/handbook/blob/master/GitHub-Adobe-Org-Management.md#request-access-to-our-adobe-github-org) to be recognized as an Adobe Employee.\n\n If you have any questions, contact Adobe\'s Open Source Office by mentioning them on the pull request with **@adobe/open-source-office** or via email <grp-opensourceoffice@adobe.com>.'
```

Comment:
Should we do "Instead add yourself to this GitHub Org", or "Instead add yourself to the Adobe GitHub Org" ?

Reply:
sure. We could do that. And update the handbook instructions to tell them to set their membership to public in the org

Reply:
Hmmm, actually, that's a good point... i forgot about all that.

Reply:
Lol now I'm second guessing my suggestion.

Reply:
Sorry! 🙈 just do what you think is right. Now I think your original seems perfectly fine.